README.md

```diff
@@ -1,2 +1,2 @@
 # otlp-prometheus-translator
-Experimental dependency-free library to translate string from OTLP format to Prometheus format.  Resources
+Library providing API to convert OTLP metric and attribute names to respectively Prometheus metric and label names.
```

Comment:
I think having "respectively" in the middle sounds very odd to me. I'm not sure that it's not proper english, but it does not read very naturally. 
```suggestion
Library providing API to convert OTLP metric and attribute names to Prometheus metric and label names respectively.
```

Reply:
Too late I'm afraid :) I found this [example](https://dictionary.cambridge.org/dictionary/english/respectively) from dictionary.cambride.org though, which looks similar:
> Persons with secondary, short secondary, and pre-secondary education achievement reported respectively 2.4%, 4.5%, and 6.0% lower health state scores.

If you feel strongly about it, you could open a PR.